events/templates/events/create_event.html

```diff
@@ -30,9 +48,11 @@ <h2 class="card-title">Event creation form</h2>
             {% endif %}
           </div>
         {% endif %}
-        <form method="post" action="/events/create/" novalidate>
+        <!-- If the user is not hosting this event, they won't be able to edit it.-->
```

Comment:
In stead of this I reckon you can just use the UserPassesTest mixin to only allow people to view the page if there the host. If they aren't it will return a 403. There should be some examples in the develop branch now (i.e Cancel event view)